### PR TITLE
[ZEPPELIN-5779] Remove pig interpreter

### PR DESCRIPTION
**Pull request by HYEEWON**
### What is this PR for?
* Remove pig interpreter

### What type of PR is it?
Refactoring

### Todos
* Remove pig and related things

### What is the Jira issue?
[ZEPPELIN-5779](https://issues.apache.org/jira/browse/ZEPPELIN-5779)

### How should this be tested?
* CI

### Questions:
* Does the licenses files need to update? Yes, Edited
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes, Edited

### CODE REVIEW

File: zeppelin-web/karma.conf.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ module.exports = function(config) {
       'bower_components/ace-builds/src-noconflict/keybinding-emacs.js',
       'bower_components/ace-builds/src-noconflict/ext-language_tools.js',
       'bower_components/ace-builds/src-noconflict/theme-chrome.js',
+      'bower_components/ace-builds/src-noconflict/mode-javascript.js',
```
Comment (jongyoul) [inline zeppelin-web/karma.conf.js:70]: Is this related to this PR? Otherwise, could you please explain what it is?

**Review by jongyoul — APPROVED** [review]

File: zeppelin-web/karma.conf.js (hunk 1 above)
Comment (HYEEWON) [reply zeppelin-web/karma.conf.js:70]: It is not related with removing pig. It was automatically added when building Zeppelin, so I just included in the pr.

Comment (jongyoul) [reply zeppelin-web/karma.conf.js:70]: It doesn't seem to make any issue but it's a bit weird that it was added while building it. Could you please check it again? If it does, please let me know how to reproduce it.

Comment (HYEEWON) [reply zeppelin-web/karma.conf.js:70]: I tested on Windows and Linux, the result was same.
1. Clone apache/zeppelin
2. Build with command 'mvn clean package -DskipTests -Phadoop2 -X'

capture
